instagram/src/components/Login/LoginPage.js

```diff
@@ -0,0 +1,78 @@
+import React from "react" 
+
+class LoginPage extends React.Component {
+  constructor () {
+    super ();
+    this.state = {
+      usernameInput: "",
+      passwordInput: "",
+    }
+  };
+
+  login = event => {
+    event.preventDefault();
+    localStorage.setItem("username", this.state.usernameInput);
+    window.location.reload()
+  }
+
+
+
+  // toggle = e => {
+  //   e.preventDefault();
+  //   if (localStorage.getItem("showFirst")) {
+  //     localStorage.removeItem("showFirst");
+  //     this.setState({
+  //       showFirst: false
+  //     });
+  //   } else {
+  //     localStorage.setItem("showFirst", true);
+  //     this.setState({
+  //       showFirst: true
+  //     });
+  //   }
+  // };
+
+  // onUsernameInput = (event) => {
+  //   this.setState({
+  //     usernameInput: event.target.value,
+  //   })
+  // };
+
+  // onPasswordInput = (event) => {
+  //   this.setState({
+  //     passwordInput: event.target.value,
+  //   })
+  // };
+
+  handleInputChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+
+  render(){
+  return (
+      <form onSubmit={this.login}>
+        <input 
+          name="usernameInput"
+          type="text" 
+          placeholder="username"
+          onChange={this.handleInputChange}
+          value={this.state.usernameInput}
+        />
+        <input 
+          name="passwordInput"
+          type="text" 
```

Comment:
when I type into your input, I can see what is being typed. That's not safe!  Change type for the password input to the following: 
type="password"